hledger-lib/Hledger/Reports/MultiBalanceReport.hs

```diff
@@ -38,55 +34,12 @@ import Hledger.Query
 import Hledger.Utils
 import Hledger.Read (mamountp')
 import Hledger.Reports.ReportOptions
+import Hledger.Reports.ReportTypes
 import Hledger.Reports.BalanceReport
 
 
--- | A multi balance report is a balance report with multiple columns,
```

Comment:
Not in so many words. I think this can basically be copied over to PeriodicReport, as almost all of the description now applies there. An abridged version just describing the specifics for MultiBalanceReport can be left here, or would you like the full description also left here for reference?

Reply:
Abridged is fine. I haven't reread it in detail, just try to save any info that looks useful to a future contributor.

Reply:
Documentation added